presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/BenchmarkReorderJoins.java

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
the case here is linear joins: `t1 join t2 join t3 join t4 ... tn`. Reorder joins will try partitions like: `t1, t3, t5` where cross join are inevitable and will iterate through them